packages/grid/x-data-grid/src/internals/models/params/gridRowParams.ts

```diff
@@ -29,7 +29,21 @@ export interface GridRowParams<R extends GridRowModel = GridRowModel> {
 }
 
 /**
- * Object passed as parameter in the row getRowHeight callback.
+ * Object passed as parameter in the row `getRowClassName` callback prop.
```

Comment:
Can we do something like:
```
interface GridRowPossitionVisibility {
  /**
   * Whether this row is the first visible or not.
   */
  isFirstVisible: boolean;
  /**
   * Whether this row is the last visible or not.
   */
  isLastVisible: boolean;
}
```
and then use that to build the 2 new interfaces?

Reply:
Done.